utils/install.py

```diff
@@ -373,6 +375,8 @@ def compare(self, version2, separator=". |-", ignorecase=True):
     "manylinux2014" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "manylinux2014" + "aarch64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "ubuntu20.04" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
+    "ubuntu20.04" + "x86_64" + WASM_BPF: VersionString("0.13.0"),
```

Comment:
We plan to ship this PR in the 0.13.2 release. Please update the `VersionString`.

Reply:
Updated now